### PR TITLE
GPS Waypoint Pulse and Custom Names

### PR DESCRIPTION
**Pull request by Azrun**
[QoL]
## About the PR 

- Adds animation to GPS verb to improve visibility and indicate a direction of spacemen get distracted.

gps_sweet_glow

- Allow for custom name for `/obj/landmark/gps_waypoint` such as those used cog2.
  - "Bar", "North Escape Wing", "South Escape Wing"
 
Non-GPS Marker Changes:
- Clarion
  - Converted pool room portion of fitness to `/area/station/crew_quarters/pool` (Pool Room)

- Horizon:
  - Reassigned hallway leading to Escape off of East to `/area/station/hallway/secondary/exit` “Escape Shuttle Hallway”

## Why's this needed? 
Improves visibility pathway depicted by GPS Verb.

```
(u)Azrun:
(*)Improved visibility of GPS line. Available in Commands window via the GPS verb. 
```

### CODE REVIEW

File: code/WorkInProgress/SomepotatosStuff.dm [hunk 1]
```diff
@@ -106,10 +133,25 @@
...
 			img.alpha=0
 			var/matrix/xf = matrix()
 			img.transform = xf/2
-			animate(img,alpha=255,transform=xf,time=2)
+			animate(img, alpha= 255, transform=xf,time=2)
+
+			img2.alpha = 0
+			animate(img2, time = 0.5 SECONDS, loop=-1 ) //noop to delay for img and maintain illuminated state
+			animate(alpha = 0, time = 2 SECONDS, loop=-1, easing = CUBIC_EASING| EASE_OUT )
+			animate(alpha = 80, time = 1 SECONDS, loop=-1, easing = BACK_EASING | EASE_OUT )
```
Comment (ZeWaka) [inline code/WorkInProgress/SomepotatosStuff.dm:154]:
```suggestion
			animate(img2, time = 0.5 SECONDS, loop = -1 ) //noop to delay for img and maintain illuminated state
			animate(alpha = 0, time = 2 SECONDS, loop = -1, easing = CUBIC_EASING | EASE_OUT )
			animate(alpha = 80, time = 1 SECONDS, loop = -1, easing = BACK_EASING | EASE_OUT )
```

**Comment by Azrun** [comment]
Waypoints Per Map





<details>
  <summary>Clarion</summary>
 
* Converted pool room portion of fitness to `/area/station/crew_quarters/pool` (Pool Room)

List is now:
AI Chamber
Arcade
Artifact Lab
Aviary
Bar
Bridge
Brig
Chemistry
Cloning
Clown Hole
Community Center
Computer Core
Detective's Office
East Electrical Substation
Engineering
Entry Hallway
Escape Shuttle Hallway
EVA Storage
Fitness Room
Genetic Research
Hydroponics Bay
Janitor's Office
Locker Room
Mechanic's Lab
Medbay Lobby
Mining Magnet Control Room
Morgue
Pathology Research
Pharmacy
Pool Room
Power Transmission Laser
Quartermaster's Storage
Ranch
Robot Depot
Robotics
Science Teleporter
Technical Storage
Teleporter
Test Chamber
Thermo-Electric Generator
Tool Storage
Toxin Lab
West Electrical Substation
</details>

<details>
  <summary>Cogmap</summary>

AI Upload Chamber
Arcade
Artifact Lab
Bar
Bridge
Brig
Captain's Quarters
Central Warehouse
Chapel
Chemistry
Cloning
Computer Core
Courtroom
Crematorium
Crew Lounge
Detective's Office
Engineering
Escape Shuttle Hallway
EVA Storage
Fitness Room
Hydroponics Lobby
Janitor's Office
Janitor's Supply Closet
Jazz Lounge
Mechanic's Lab
Medbay
Medbay Lobby
Medical Research
Mining Staff Room
Morgue
Owlery
Pod Bay
Pool Room
Power Transmission Laser
Quartermaster's Office
Refinery
Robot Depot
Robotics
Science Teleporter
Security Wing
Stock Exchange
Technical Storage
Test Chamber
Tool Storage
Toxin Lab
Waste Disposal
</details>

<details>
  <summary>Cogmap2</summary>

AI Chamber
AI Upload Chamber
Arcade
Artifact Lab
Bar
Bridge
Brig
Chapel
Chemistry
Cloning
Clown's Quarters
Crew Quarters A
Emergency Solar Array 2
Engineering
EVA Storage
Fitness Room
Genetic Research
Hydroponics Bay
Janitor's Office
Janitor's Supply Closet
Mechanic's Lab
Mechanics
Medbay Lobby
Morgue
Net Cafe
North Escape Wing
North Solar Maintenance
Owlery
Pharmacy
Pod Bay
Pod Bay Security Checkpoint
Pool Room
Quartermaster's Office
Refinery
Robot Depot
Robotics
Science Teleporter
South Escape Wing
South Solar Maintenance
Technical Storage
Teleporter
Toxin Lab
</details>

<details>
  <summary>Destiny</summary>

AI Chamber
Arcade
Artifact Lab
Aviary
Bar
Bridge
Brig
Chapel
Chemistry
Cloning
Clown Hole
Computer Core
Courtroom
Engineering
Escape Shuttle Hallway
Fitness Room
Hydroponics Bay
Janitor's Office
Mechanic's Lab
Medbay Lobby
Medical Booth
Medical Research
Mining Refinery
Mining Staff Room
Morgue
Pharmacy
Pool Room
Quartermaster's Office
Robot Depot
Robotics
Science Teleporter
Security Checkpoint
Technical Storage
Teleporter
Test Chamber
Thermo-Electric Generator
Toxin Lab
Waste Disposal
</details>

<details>
  <summary>Donut3 </summary>

AI Satellite
AI Upload Chamber
Arcade
Artifact Lab
Bridge
Cafeteria
Chapel
Chemistry
Cloning
Courtroom
Crew Lounge
Detective's Office
East Solar Maintenance
Engineering
Escape Shuttle Hallway
EVA Storage
Fitness Room
Fuq 3 clothing store
Genetic Research
Hydroponics Bay
Janitor's Office
Library
Mechanic's Lab
Medbay Lobby
Morgue
Pharmacy
Pod Bay
Power Transmission Laser
Quartermaster's Office
Ranch
Refinery
Robot Depot
Robotics
Science Teleporter
Security
Singularity Core
Stock Exchange
Technical Storage
Tool Storage
Toxin Lab
West Solar Maintenance
</details>

<details>
  <summary>Horizon</summary>

* Reassigned hallway leading to Escape off of East to `/area/station/hallway/secondary/exit` “Escape Shuttle Hallway”

AI Chamber
Arcade
Bar
Bridge
Chapel
Chemistry
Cloning
Clown Hole
Computer Core
Courtroom
Engineering Control Room
Escape Shuttle Hallway
EVA Storage
Fitness Room
Genetic Research
Head of Personnel's Office
Hydroponics Bay
Janitor's Office
Mechanic's Lab
Medbay Lobby
Medical Booth
Mining Refinery
Mining Staff Room
Net Cafe
Owlery
Pharmacy
Pool Room
Quartermaster's Office
Ranch
Robotics
Science Lobby
Science Teleporter
Security Foyer Checkpoint
South Crew Quarters
South Solar Maintenance
Technical Storage
Teleporter
Tool Storage
</details>

<details>
  <summary>Oshan</summary>

AI Chamber
Arcade
Artifact Lab
Bar
Chapel
Chemistry
Cloning
Clown Hole
Computer Core
Courtroom
Customs
Detective's Office
Engineering
Engineering EVA Storage
Escape Shuttle Hallway
EVA Storage
Fitness Room
Genetic Research
Hydroponics Bay
Janitor's Office
Janitor's Supply Closet
Kitchen
Mechanic's Lab
Medbay Lobby
Medical Director's Office
Merchant Dock
Net Cafe
Pharmacy
Psychiatrist's Office
Quartermaster's Office
Radio Lab
Ranch
Refinery
Robot Depot
Robotics
Science Teleporter
Sea Elevator Room
Security Foyer Checkpoint
Supply Lobby
Technical Storage
Teleporter
Toxin Storage
Zen Garden
</details>

<details>
  <summary>Kondaru </summary>

AI Chamber
AI Upload Chamber
Arcade
Artifact Lab
Aviary
Barber Shop
Bridge
Brig
Cafeteria
Central Warehouse
Chapel
Chemistry
Cloning
Clowntainment
Computer Core
Courtroom
Customs Security Checkpoint
Detective's Office
Diplomatic Quarters
East Solar Maintenance
Emergency Storage A
Engineering
Escape Hallway Security Checkpoint
EVA Storage
Fitness Room
Genetic Research
Hydroponics Bay
Interrogation Room
Janitor's Office
Janitor's Supply Closet
Kitchen
Mechanic's Lab
Medbay Lobby
Medical Booth
Mining Refinery
Mining Staff Room
Morgue
Nerd Dungeon
News Office
North Crew Quarters
North Escape Wing
Pathology Research
Pharmacy
Pod Bay
Pod Bay Security Checkpoint
Pool Room
Power Transmission Laser
Public Market
Quartermaster's Office
Ranch
Research Router
Robot Depot
Robotics
Sauna
Science Lobby
Science Teleporter
Security
South Crew Quarters
South Escape Wing
Technical Storage
Teleporter
Thermo-Electric Generator
Toilets
Toxin Lab
Waste Disposal
West Solar Maintenance
</details>

<details>
  <summary>Atlas </summary>

Armory
Artifact Lab
Bridge
Cafeteria
Catering Storage
Chapel
Chapel Office
Chemistry
Chief Engineer's Office
Cloning
Clown Hole
Combustion Chamber
Computer Core
Crew Lounge
Cryogenic Crew Storage
Customs
Detective's Office
East Electrical Substation
Engineering
Escape Shuttle Hallway
EVA Storage
Genpop Cell
Head of Personnel's Office
Hydroponics Bay
Janitor's Office
Kitchen
Mechanic's Lab
Medbay
Medical Director's Office
Medical Research
Mining Refinery
Mining Staff Room
Morgue
Owlery
Pathology Research
Quartermaster's Office
Ranch
Research Director's Office
Research Dock
Robotics
Science Teleporter
Security
Technical Storage
Teleporter
Thermo-Electric Generator
Toilets
Tool Storage
Toxin Lab
Toxin Storage
West Electrical Substation
</details>

<details>
  <summary>Manta</summary>

AI Upload Chamber
Arcade
Artifact Lab
Bridge
Brig
Cargo Bay
Chapel
Chemistry
Cloning
Clown Hole
Communications Office
Courtroom
Cryogenic Crew Storage
Engineering Power Room
Escape
Fitness Room
Hydroponics Bay
Janitor's Office
Mechanic's Lab
Medbay Lobby
Medical Research
Pathology Research
Port Torpedo Bay
Ranch
Robot Depot
Robotics
Science Teleporter
Singularity Core
Starboard Torpedo Bay
Technical Storage
Teleporter
Test Chamber
The Rising Tide
Waste Disposal
</details>

File: code/WorkInProgress/SomepotatosStuff.dm [hunk 2]
```diff
@@ -41,10 +41,21 @@
 			boutput( user, "<span class='notice'>You attach the [CLEAN(src)] to the [CLEAN(what)].</span>" )
 		*/
 
+//#define GPS_MAP_TESTING
```
Comment (ZeWaka) [inline code/WorkInProgress/SomepotatosStuff.dm:44]: needed? if so, should comment on what it does

Comment (Azrun) [reply code/WorkInProgress/SomepotatosStuff.dm:49]: 📝DOCS 📝

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**